src/Sulu/Bundle/MediaBundle/Tests/Functional/Entity/MediaDataProviderRepositoryTest.php

```diff
@@ -214,7 +212,7 @@ private function createMedia($title, $collection, $mimeType, $type, $tags = [],
         return $media;
     }
 
-    private function createRole($name, $system)
+    private function createRole($name, $system): Role
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes